### ISSUE TITLE
Kebechet update are throwing type error in workflow

### ISSUE THREAD

**Issue by harshad16** [thoth-station/kebechet#755]
**Describe the bug**
The workflows are throwing the type error for the kebechet updates.

version: v1.3.2 

```
Traceback (most recent call last):
  File "/home/user/kebechet/kebechet_runners.py", line 180, in run
    instance.run(**manager_configuration)
  File "/home/user/kebechet/managers/version/version.py", line 420, in run
    with cloned_repo(self) as repo:
  File "/usr/lib64/python3.8/contextlib.py", line 113, in __enter__
    return next(self.gen)
  File "/home/user/kebechet/utils.py", line 68, in cloned_repo
    access_token = manager.service.authentication.get_token()
TypeError: get_token() missing 2 required positional arguments: 'namespace' and 'repo'
```

**Comment by KPostOffice** [comment]
Fixed by https://github.com/thoth-station/kebechet/pull/744

**Comment by goern** [comment]
> Fixed by #744

/close
